crates/wasmtime/src/runtime/externals/global.rs

```diff
@@ -130,6 +130,9 @@ impl Global {
                                 })
                                 .into(),
                         ),
+                        HeapType::NoCont | HeapType::ConcreteCont(_) | HeapType::Cont => todo!(
+                            "Embedder support for continuations has not yet been implemented!"
+                        ),
```

Comment:
This location unfortunately can't easily be switched to returning a `Result` so panicking makes sense to me, but mind tagging this with `// FIXME(...)` pointing to an issue?